src/_settings.scss

```diff
@@ -265,8 +265,7 @@ $fd-width--gutter: map-get($fd-spacing, s); // desktop default
 
 // Form Configuration
 $fd-forms-height: $fd-spacing--base * 9 !default; // 36px
-$fd-forms-height--compact: $fd-spacing--base * 7 !default; // 28px
-$fd-forms-height--input-text: $fd-forms-height !default; // deprecate
+$fd-forms-height--compact: $fd-spacing--base * 6.5 !default; // 26px before 28px
```

Comment:
This is also used in `fd-form-select` class (native select component). Should it be changed that way? I saw that we have button configuration below (278-281 lines)
![image](https://user-images.githubusercontent.com/26483208/70995425-d2320000-20d0-11ea-9775-50a7cb83a4ec.png)

Reply:
I thin so, You can check the message https://github.com/SAP/fundamental-styles/issues/417#issuecomment-550999964

Reply:
I think Jedrzej is right. Changing the form-height affects all forms and hence all the tests that are updated. The best way to "solve" this issue is to create new variables where you can declare the button height in cosy and compact mode

Reply:
done, the default value was restored and new variable inside the button file introduced